js/controller.js

```diff
@@ -635,6 +641,41 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.state.mainVideoDuration = this.state.duration;
       }
     },
+  
+    onTouching: function(params) {
+      if (this.videoVr) {
+        this.mb.publish(OO.EVENTS.TOUCHING, this.focusedElement, params);
+      }
+    },
+  
+    onTouched: function () {
+      if (this.videoVr) {
+        this.mb.publish(OO.EVENTS.TOUCHED, this.focusedElement);
+      }
+    },
+  
+    setViewingDirection: function(event, yaw, roll, pitch) {
+      this.state.viewingDirection = {yaw: yaw, roll: roll, pitch: pitch};
+    },
+  
+    recreatingUI: function (event, elementId, params, settings) {
+      console.log('call recreatingUI args', arguments);
```

Comment:
Is this leftover debug code, or is it useful to have all the time?  Maybe put this in a debugVR check.